Makefile

```diff
@@ -189,8 +189,227 @@ bedrock-markdown-links:
 
 install-geth:
 	./ops/scripts/geth-version-checker.sh && \
-	 	(echo "Geth versions match, not installing geth..."; true) || \
- 		(echo "Versions do not match, installing geth!"; \
- 			go install -v github.com/ethereum/go-ethereum/cmd/geth@$(shell cat .gethrc); \
- 			echo "Installed geth!"; true)
+	(echo "Geth versions match, not installing geth..."; true) || \
+	(echo "Versions do not match, installing geth!"; \
+	go install -v github.com/ethereum/go-ethereum/cmd/geth@$(shell cat .gethrc); \
+	echo "Installed geth!"; true)
 .PHONY: install-geth
+
+### Rootstock ###
+
+rsk-regtest-log:
+	docker exec -ti -w "/var/lib/rsk/logs" rsk_regtest /bin/bash -c "tail -f rsk.log"
+.PHONY: rsk-regtest-log
+
+rsk-regtest-up:
+	docker start rsk_regtest && \
+	make rsk-regtest-log
+.PHONY: rsk-regtest-up
+
+rsk-regtest-down:
+	docker stop rsk_regtest
+.PHONY: rsk-regtest-up
+
+rsk-regtest-stop:
+	docker stop rsk_regtest
+.PHONY: rsk-regtest-stop
+
+rsk-regtest-delete:
+	docker rm rsk_regtest
+.PHONY: rsk-regtest-delete
+
+rsk-regtest-start:
+	docker run --name rsk_regtest -d -it \
+	-p 5050:5050 -p 4444:4444 -p 4445:4445 \
+  -v $(PWD)/rskj.logback.xml:/etc/rsk/logback.xml \
+  -v $(PWD)/rskj.node.conf:/etc/rsk/node.conf \
+  franciscotobar/rskj:optimism-6.1.0 --regtest --reset
```

Comment:
if I understood correctly, we don't need the custom rskj image any longer
```suggestion
  rsksmart/rskj:FINGERROOT-5.3.0 --regtest --reset
```
or 
```suggestion
  rsksmart/rskj:latest --regtest --reset
```

Reply:
we don't?

Reply:
will try to remove and test it along with the price cap enabled

Reply:
done.
ps: price cap remains disabled as we need to change the create2 transaction before we can enable it again. See https://rsklabs.atlassian.net/browse/ROLLUP-528

Reply:
turns out this is still necessary for, at least, the proposer:
![Screenshot 2024-01-22 at 15 19 51](https://github.com/rsksmart/optimism/assets/36888576/aa7bacc4-56bc-40fc-bec7-93de50d406f3)

Reply:
I dont think that is related with my change, I think that this is related with other step. Ill need to double-check it.

Reply:
well, please do by all means, but just to let you know that I see this after making the only change of using the official image instead of yours.

Reply:
ok, I'm going to keep it as is, considering the aim of this task.